docs/src/1_Mechanics/06_Bestiary.yaml

```diff
@@ -295,17 +310,59 @@ Ritualist, Support:
     - Shield, Others
   Description: A Support with access to Powers.
 
-Specter:
-  Type: NPC
-  HP: 3
-  AP: 0
+King Candy: 
+  Type: Dealer
+  Level: 2
+  Threat: 2
+  HP: 10
+  AP: 1
   AR: 2
-  PP: 2
-  Attrib:
+  Attribs:
+    AGL: -1
     GUT: 1
+    CON: 1
     INT: 1
     STR: -1
+    VIT: -1
   Powers:
     - Attack, Mystic
     - Attack, Mystic Aura
-    - Slow
+    - Attack, Mystic Entangle
+    - Shield, Self
+    - Teleport Minions (1x target)
+    - Shield, Self
+  House Actions:
+  - In House:
+      - Summon: Summon 2x minions
+      - Entangle: Entangle 2x targets
+      - Mind control: 2 uses, only if 'primed' - grappled or entangled
+  - In Lake:  
+      - Clubs: Lake attempts to grab every creature currently in the lake. DR 2 save, on failure entangled and dragged 5 feet under water.
+      - Diamonds: Fissure appears in the ground in a determined quadrant. DR 2 save, on failure entangled. 
+      - Spades: Force push on a space determined by GM. Pushed 5 spaces back from push, DR 2 save, on failure take 1 damage. 
+      - Hearts: Sugar crystals spray from the lake, targeting anyone within 2 spaces of the lake shore. DR 2 save, on failure take 2 damage. On success, half damage.
```

Comment:
We had previously discussed House Actions as one type of Power exclusive to Bosses. Separating them from powers here implies a different organization structure. 

Similarly, teleport minions above is not implemented as a power

Reply:
As long the end user sees House Actions as separate from Powers. Does that complicate the code to want it to print as separate boxes but write it as the same type?

Reply:
There's the organization presented to outside viewers (e.g., Powers = major+minor+passive, separate from Vulnys, etc) and the organization we use internally (e.g., Powers are major+minor+vulnys, etc). In general, it's best to align these as it simplifies code, but it's not hard to pull a subset of a list based on a type